src/client/components/features/LesEntreprisesSEngagent/MonEspace/MonEspaceEntreprise.module.scss

```
@@ -1,85 +1,80 @@
 @use "@styles/utilities-deprecated";
+@use "@styles/utilities";
+
+$background-color: utilities.$color-background-primary-alternative;
 
 .monEspace {
   margin: 0 auto;
-  padding: 2rem 1rem 2rem 1rem;
-  background-color: utilities-deprecated.$color-background-white-lilac;
-
-  &Titre {
-    color: utilities-deprecated.$color-primary;
-    font-size: 1.5rem;
-    font-weight: 700;
-    line-height: 1.75rem;
-    margin-bottom: 1rem;
+  padding: 1rem;
+  background-color: $background-color;
+  @include utilities.media(large) {
+    padding: 2rem 4rem;
   }
 
-  &Accroche {
-    color: utilities-deprecated.$color-secondary;
-    font-size: 1.5rem;
-    font-weight: 400;
-    margin-bottom: 1.25rem;
-    line-height: 1.75rem;
+  span {
+    display: block;
   }
 
-  &Description {
-    text-align: left;
-    margin-bottom: 1.5rem;
-  }
+  h1 {
+    margin-bottom: 2rem;
+    @extend %text-xLarge;
 
-  &Connexion {
-    margin-bottom: 1.5rem;
-    width: 100%;
-    justify-content: center;
+    @include utilities.media(large) {
+      font-weight: bold;
+      text-align: center;
+    }
   }
-}
 
-@include utilities-deprecated.media(large) {
-  .monEspace {
-    padding: 4rem 1rem 4rem 1rem;
+  & .entreprisesPourLaJeunesse {
+    display: grid;
+    grid-auto-rows: auto;
+    grid-row-gap: 1.5rem;
+
+    @include utilities.media(large) {
+      grid-auto-flow: column;
+      grid-template-columns: 1fr 1fr;
+      grid-template-rows: auto auto auto auto;
+      grid-column-gap: 5rem;
+    }
 
-    .container {
-      max-width: 45rem;
+    & img {
       margin: auto;
+      height: 100%;
+      padding: 1rem;
+    }
 
-      &Boutons {
-        display: flex;
-        flex-direction: column;
-        max-width: 28rem;
-        margin: 40px auto auto;
-      }
+    & h2 {
+      @extend %text-large;
+      text-align: center;
+    }
 
-      &Info {
-        max-width: 30rem;
-        margin: 40px auto auto;
-        display: flex;
-        flex-direction: column;
-        gap: 1rem
+    & p {
+      @extend %text-regular;
+
+      span {
+        margin-top: 0.5ch;
       }
     }
 
-    &Titre {
-      color: utilities-deprecated.$color-primary;
-      font-weight: 700;
-      text-align: left;
-    }
+    & a {
+      width: 100%;
 
-    &Accroche {
-      font-weight: 400;
-      color: utilities-deprecated.$color-secondary;
-      margin-bottom: 3.5rem;
-      text-align: left;
+      &:last-child:not(:first-child) {
```

```suggestion
      &:not(:only-child):last-child {
```
Par contre, je ne trouve pas où c'est utilisé 🤔